packages/openid4vc/src/openid4vc-holder/OpenId4VciHolderService.ts

```diff
@@ -412,10 +470,24 @@ export class OpenId4VciHolderService {
         .withToken(tokenResponse.access_token)
 
       const credentialRequestClient = credentialRequestBuilder.build()
+
+      let createDPoPOptions: CreateDPoPClientOptions | undefined
+      if (options.dPoPJwk) {
```

Comment:
Isn't the dpop jwk also included in the access token cnf? In that case, can't we dynamically detect it and then use dpop?

Reply:
no

Reply:
Interesting. For sd-jwt they include the jwk directly but here a fingerpring...